github/pull_request.rb

```diff
@@ -105,6 +113,15 @@ def check_commits_style
     end
   end
 
+  def add_issue_links
+    if new? && issue_numbers.any?
+      message = issue_numbers.inject("\nIssues:") do |msg, issue_number|
+        msg + " [##{issue_number}](http://projects.theforeman.org/issues/#{issue_number})"
+      end
+      append_pull_request_body(message)
```

Comment:
So this does not add a fresh comment? How does the output on a PR come out to look like if this is updating the issue? Are we altering the users first comment ?